src/bundler/server.rs

```diff
@@ -0,0 +1,35 @@
+use async_trait::async_trait;
+use tonic::Response;
+
+use crate::uopool::server::{
+    bundler::{bundler_server::Bundler as BundlerServer, SetModeRequest, SetModeResponse},
+    types::{GetChainIdResponse, GetSupportedEntryPointsResponse},
+};
+
+use super::Bundler;
+
+#[async_trait]
+impl BundlerServer for Bundler {
```

Comment:
Do you think it make sense to add BundlerService (similar to how it is done for user operation mempool)? Or will this just add not needed complexity?

Reply:
I am not sure I understand your questions here.

Do you mean we should remove this https://github.com/Vid201/aa-bundler/blob/main/src/proto/bundler/bundler.proto ?

Reply:
No. This is now solved, my idea was to add `BundlerManager`, like you already did (we can rename it to BundlerService to it will have same name as `UoPoolService` - https://github.com/Vid201/aa-bundler/blob/main/src/uopool/services/uopool.rs#L86).